test/contrib/rails/controller_test.rb

```diff
@@ -21,12 +21,9 @@ class TracingControllerTest < ActionController::TestCase
     assert_equal(spans.length, 2)
 
     span = spans[1]
-    assert_equal(span.name, 'rails.request')
+    assert_equal(span.name, 'rails.action_controller')
     assert_equal(span.span_type, 'http')
     assert_equal(span.resource, 'TracingController#index')
```

Comment:
So, as I understand it, all this http metadata has been bounced to rack, which is actually handling the protocol and should carry the now missing info in its spans, right?